### ISSUE TITLE
Complain when log messages are capitalized

### ISSUE THREAD

**Issue by xaprb** [golang/lint#41]
I'd like to suggest that log messages should be lowercase similar to error messages, e.g.

``` go
log.Println("This is a log message")
```

should probably be lowercased.
